.travis.yml

```diff
@@ -34,6 +36,14 @@ matrix:
       env: PUPPET_GEM_VERSION="~> 3.3.0"
     - rvm: 2.1.0
       env: PUPPET_GEM_VERSION="~> 3.4.0"
+    - rvm: 1.8.7
+      env: PUPPET_GEM_VERSION="~> 4.2.0"
+    - rvm: 1.8.7
+      env: PUPPET_GEM_VERSION="~> 4.3.0"
```

Comment:
Found that I was actually only testing 4 and 4 STRICT_VARIABLES="yes". I've added 4.0.0 to 4.4.0 to the tests now.